### PR TITLE
Adding some docs and basic test

### PR DESCRIPTION
**Pull request by rcarroll380**

### CODE REVIEW

File: test/services/chargeServiceTest.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const assert = require('assert');
+const chargeService = require('../../api/services/chargeService');
+
+describe('chargeService', () => {
+  describe('search', () => {
```
Comment (cookmn) [inline test/services/chargeServiceTest.js]: Shouldn't this be 'createCharge', not 'search'?

Comment (rcarroll380) [reply test/services/chargeServiceTest.js]: Yep. just caught that in my next branch. Will fix it there.

File: api/services/chargeService.js [hunk 2]
```diff
@@ -1,9 +1,24 @@
 'use strict';
 
+const _ = require('lodash');
+
+function createCharge() {
+  const chargeObj = {
+    amount: _.sample([100, 250, 500, 750, 1000]),
+    date: '1/1/2018',
```
Comment (EPICmynamesBG) [inline api/services/chargeService.js]: Do we want to randomize the dates too? :D `moment().subtract(_.random(0, 100), 'days')).format()`?

**Comment by rcarroll380** [comment]
@EPICmynamesBG yes to both. patience grasshopper. those are coming soon. trying to do small PRs vs 1 giant one.

File: test/services/chargeServiceTest.js (hunk 1 above)
Comment (dustinwaldron) [reply test/services/chargeServiceTest.js]: @rcarroll380 do you need to add a test for the `build` function or will that be later too?

**Pull request merged**